# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1311868778 14400
#      Thu Jul 28 11:59:38 2011 -0400
# Branch stable
# Node ID c68fe26745aeb046d3c5adb7b2fac4342a63ce70
# Parent  a19b50f6697f62b03441ff03eb31e607aa15c723
maint: include fclose and stdio in bootstrap.conf module list

* bootstrap.conf (gnulib_modules): Include fclose and stdio
in the list.

diff --git a/build-aux/bootstrap.conf b/build-aux/bootstrap.conf
--- a/build-aux/bootstrap.conf
+++ b/build-aux/bootstrap.conf
@@ -16,16 +16,17 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 
 # gnulib modules used by this package.
 gnulib_modules="
   c-strcase
   copysign
   crypto/md5
+  fclose
   fcntl
   filemode
   fnmatch
   getcwd
   gethostname
   getopt-gnu
   gettimeofday
   glob
@@ -46,16 +47,17 @@ gnulib_modules="
   roundf
   sigaction
   signal
   sigprocmask
   sleep
   sleep
   stat
   stdint
+  stdio
   strftime
   strptime
   symlink
   sys_stat
   sys_time
   sys_times
   time
   times
